js/libs/keycloak-js/src/keycloak.js

```diff
@@ -1021,27 +1022,11 @@ function Keycloak (config) {
     }
 
     function decodeToken(str) {
```

Comment:
I think we can replace this entire function with `jwtDecode`, there should be no need to keep the exception the same here.